puzzle-utils/words/crossword_clue.py

```diff
@@ -3,19 +3,40 @@
 import urllib.parse
 from bs4 import BeautifulSoup
 
+def crossword_clue_multi(clues, solver = "wordplays"):
+    """
```

Comment:
Have you considered using `*args` and `**kwargs` to simplify how this method can be called?

Reply:
I'll switch to args in my next review where I'll be removing the solver parameter anyways.